app/views/bookings/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Hello are you working?</h1>
```

Comment:
Is this really necessary? I get that we don't have any data show for now, but maybe add comments for what we need to do to display bookings.